helm_deploy/hmpps-person-record/values.yaml

```diff
@@ -58,10 +58,6 @@ generic-service:
       HMPPS_SQS_QUEUES_CPRDELIUSOFFENDEREVENTSQUEUE_QUEUE_NAME: "sqs_queue_name"
     sqs-cpr-delius-offender-events-dlq-secret:
       HMPPS_SQS_QUEUES_CPRDELIUSOFFENDEREVENTSQUEUE_DLQ_NAME: "sqs_queue_name"
-    sqs-cpr-offender-events-queue:
-      HMPPS_SQS_QUEUES_CPROFFENDEREVENTSQUEUE_QUEUE_NAME: "sqs_queue_name"
-    sqs-cpr-offender-events-dlq:
-      HMPPS_SQS_QUEUES_CPROFFENDEREVENTSQUEUE_DLQ_NAME: "sqs_queue_name"
```

Comment:
This is to stop the noise in the alert channel. 

`Caused by: software.amazon.awssdk.services.sqs.model.QueueDoesNotExistException: The specified queue does not exist or you do not have access to it. (Service: Sqs, Status Code: 400, Request ID: 878707d8-ca08-5648-88be-59f57eeb03e6)`